android/assets/jsons/translations/template.properties

```diff
@@ -1127,3 +1127,7 @@ in all cities with a world wonder =
 in all cities connected to capital = 
 in all cities with a garrison = 
 
+# Uniques not found in JSON files
+
+Only available after [] turns = 
+This Unit upgrades for free = 
```

Comment:
You sure the last translation template works without a linefeed at the end of the file?